modules/images-configuration-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,133 @@
+ - Pod CIDR:                <pod_cidr>
+ - Host Prefix:             /23
+ - Subnets:                 <subnet_ids>
+EC2 Metadata Http Tokens:   optional
+Role (STS) ARN:             arn:aws:iam::<AWS_id>:role/<account_roles_prefix>-HCP-ROSA-Installer-Role
+Support Role ARN:           arn:aws:iam::<AWS_id>:role/<account_roles_prefix>-HCP-ROSA-Support-Role
+Instance IAM Roles:
+ - Worker:                  arn:aws:iam::<AWS_id>:role/<account_roles_prefix>-HCP-ROSA-Worker-Role
+Operator IAM Roles:
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-kube-system-capa-controller-manager
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-kube-system-control-plane-operator
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-kube-system-kms-provider
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-openshift-image-registry-installer-cloud-cred
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-openshift-ingress-operator-cloud-credentials
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-openshift-cluster-csi-drivers-ebs-cloud-credent
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-openshift-cloud-network-config-controller-cloud
+Managed Policies:           Yes
+State:                      ready
+Private:                    No
+Delete Protection:          Disabled
+Created:                    Oct 01 2030 09:48:52 UTC
+User Workload Monitoring:   Enabled
+OIDC Endpoint URL:          https://<endpoint> (Managed)
+Audit Log Forwarding:       Disabled
+External Authentication:    Disabled
+Etcd Encryption:            Disabled
+Registry Configuration:
+ - Allowed Registries:     <allowed_registry> <1>
+ - Insecure Registries:    <insecure_registry> <2>
+ - Allowed Registries for Import: <3>
+    - Domain Name:          <domain_name> <4>
+    - Insecure:             true <5>
+----
+<1> `Allowed Registries`: A comma-separated list of registries for which image pull and push actions are allowed.
+<2> `Insecure Registries`: A comma-separated list of registries which do not have a valid TLS certificate or only support HTTP connections.
+<3> `Allowed Registries for Import`: Limits the container image registries from which normal users can import images. The format should be a comma-separated list of `domainName:insecure`.
+<4> `domainName`: Specifies a domain name for the registry.
+<5> `insecure`: Indicates whether the registry is secure or insecure.
+
+. To check that the changes are applied, list your nodes:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                         STATUS                     ROLES                  AGE   VERSION
+ip-10-0-137-182.us-east-2.compute.internal   Ready,SchedulingDisabled   worker                 65m   v1.30.3
+ip-10-0-139-120.us-east-2.compute.internal   Ready,SchedulingDisabled   control-plane          74m   v1.30.3
+ip-10-0-176-102.us-east-2.compute.internal   Ready                      control-plane          75m   v1.30.3
```

Comment:
remove line

Reply:
removed the 3 lines relating to the control-plane